liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/clearChecksums.test.groovy

```diff
@@ -28,33 +34,6 @@ Optional Args:
     Default: null
 """
 
-    run "Happy path", {
-        arguments = [
-            url:        { it.url },
-            username:   { it.username },
-            password:   { it.password }
-        ]
-        setup {
-            history = [
-                    new HistoryEntry(
-                            id: "1",
-                            author: "test",
-                            path: "com/example/changelog.xml"
-                    ),
-                    new HistoryEntry(
-                            id: "2",
-                            author: "test",
-                            path: "com/example/changelog.xml"
-                    ),
-            ]
-        }
-
-        expectedResults = [
-                statusCode   : 0
-        ]
-
-    }
-
```

Comment:
Why was this test removed?

Reply:
This test was failing because it couldn't find any expected result. I did change it by removing `expectedResults` object and added an expectation one with the logic of verifying each deployed changeset had checksum field set as `NULL`, but that's exactly the same check I had first implemented in `"validate checksums are cleared"()`  from **ClearChecksSumsIntegrationTest**. I thought I can remove that scenario as it was already covered by the integration test.

Reply:
Works for me.